packages/client/src/context/Auth.context.tsx

```diff
@@ -106,28 +116,59 @@ export const AuthProvider: FC<AuthProviderProps> = ({ children }) => {
     handleUnauthenticated();
   };
 
+  // Handle switch tab
+  const [activeTab, setActiveTab] = useState<string>('login');
```

Comment:
NIT (short for nit-pick): Having the type be represented slightly differently will help make sure errors from typos don't show up.

```
useState<'login' | 'signup'>('login');
```

The code above will make it so typescript will check to make sure only one of those two options are used